pkg/network/events/monitor_linux.go

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package events handles process events
+package events
+
+import (
+	"time"
+
+	"go4.org/intern"
+
+	"github.com/DataDog/datadog-agent/pkg/security/secl/model"
+)
+
+func getProcessStartTime(ev *model.Event) time.Time {
+	if ev.GetEventType() == model.ExecEventType {
+		return ev.GetProcessExecTime()
+	}
+	if ev.GetEventType() == model.ForkEventType {
+		return ev.GetProcessForkTime()
+	}
+	return time.Time{}
```

Comment:
I recognize that this block is moved from above into a dedicated function (which is good), so if we don't want to modify then that'd make sense.

I'm curious what the emergent behavior is as the `time` for this process is processed in the agent / backend if we return `time.Time{}` - what do you think about returning an error and skipping this instead? Mostly asking because if we have an EventType that we don't expect, that might be indicative of another problem and this `time.Time{}` return hides.

Reply:
So, the reason I broke it out is because linux supports two event types, fork & exec.  windows only supports exec.

The main thing this time is used for (on the windows side) is to correlate processes in the process cache.
Beyond that, objective was to preserve existing behavior.

Reply:
to say another way, my objective was to not change the behavior of the pre-existing code.  This maintains the previous behavior.